src/auto-evo/RunResults.cs

```diff
@@ -368,6 +389,129 @@ public long GetPopulationInPatch(Species species, Patch patch)
             }
         }
 
+        /// <summary>
+        ///   Computes the final population of species, by patch.
+        /// </summary>
+        /// <remarks>
+        ///   <para>
+        ///     TODO: Currently no enforced correlation with actual changes applied.
+        ///   </para>
+        ///   <para>
+        ///     Species are only registered if they population go above 0.
```

Comment:
```suggestion
        ///     Species are only returned if their population is above 0.
```
? is that what you meant?